DataAccess/Public/Include/DataAccess/FutureResult.hpp

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+namespace DataAccess
+{
+    using RawFuture = std::future<std::any>;
+
+    /**
+     * @brief   Custom future.
+     * @details Uses like common wrapper for std::future.
+     */
+    template <typename TFromAny>
+    struct FutureResult
+    {
+    private:
+        RawFuture _future;
+
+    public:
+        explicit FutureResult(RawFuture&& future) : _future(std::move(future)) {}
```

Comment:
I guess at that rule we have mistake because at the same time we have to name class fields with underscore, but it doesn't happen in the rule, but I'll fix it

Reply:
Also, rule says that we can name parameter with the same name as field only if this isn't used in the body of constructor, and parameter is used in the initialiazer list of constructor, not body
I think this rule should provide a way about how to avoid misunderstandings of naming(mostly when names are equal), but there everythink is okey